CHANGELOG.md

```diff
@@ -836,9 +835,40 @@ Non-backwards compatible changes
   Therefore, only _declarations_ of `PartialOrder` records will need their field names
   updating.
 
-* NB (issues #1214 #2098) the corresponding situation regarding the `flip`ped
-  relation symbols `_≥_`, `_>_` (and their negated versions!) has not (yet)
-  been addressed.
+* NB (issue #2098) the corresponding situation regarding the `flip`ped
+  relation symbols `_≥_`, `_>_` has not (yet) been addressed.
+
+### (Issue #1214) Reorganisation of the introduction of negated relation symbols under `Relation.Binary`
+
+* Previously, negated relation symbols `_≰_` (for `Poset`) and `_≮_` (`TotalOrder`)
+  were introduced in the corresponding `Relation.Binary.Properties` modules, for re-export.
+
+* Now they are introduced as definitions in the corresponding `Relation.Binary.Bundles`,
+  together with, for uniformity's sake, an additional negated symbol `_⋦_` for `Preorder`.
+  NB `_≮_` is now introduced *earlier* in the hierarchy, at `StrictPartialOrder`, which
+  incurs some rather complicated deprecation gymnastics...
```

Comment:
I don't think this NB is needed. The user doesn't care about "complicated deprecation gymnastics", and I don't there's anything special about `_≮_`

Reply:
Re: gymnastics, see my comment above. Because the symbol has moved, its deprecation can't simply be achived locally inside one module.
As for the user, the warning should tell them corretcly where to import form , but his note was additional reinforcement.
As for the author of the PR, I was trying to document as clearly as possible what was quite a complicated thought process while I was doing this, for the sake of subsequent reviewers ;-) and myself or anyone else trying to tackle #2098...